scipy/stats/_multivariate.py

```diff
@@ -5530,9 +5530,8 @@ def _docfill(obj, docdict, template=None):
 class uniform_direction_gen(multi_rv_generic):
     r"""A vector-valued uniform direction.
 
-    Return a random direction (unit vector).
-
-    The `dim` keyword specifies the dimension.
+    Return a random direction (unit vector). The `dim` keyword specifies
+    the dimension.
```

Comment:
I mentioned in the original PR that "dimensionality" is the [more appropriate word](https://www.quora.com/What-is-the-difference-between-dimension-dimensional-and-dimensionality), so while we're at it...
```suggestion
    the dimensionality of the space.
```